src/hamster/overview.py

```diff
@@ -538,6 +542,21 @@ def on_timeout(self):
         # The timeout will stop if returning False
         return True
 
+    def on_help_clicked(self, menu):
+        uri = "help:hamster-time-tracker"
+        try:
+            gtk.show_uri(gdk.Screen().get_default(), uri,
```

Comment:
```suggestion
            gtk.show_uri(None, uri,
```
`None` seems the standard way to say "default location"
https://lazka.github.io/pgi-docs/#Gtk-3.0/functions.html#Gtk.show_uri
https://programtalk.com/python-examples/gi.repository.Gtk.show_uri/
It does not seem possible to make a suggestion on several lines,
but of course it should be `gtk.show_uri(None, uri, gdk.CURRENT_TIME)`

Reply:
Yes, "None" works, too. It comes down to exactly the same thing AFAICT.

Reply:
If you read that documentation, you'd expect Gdk.Screen() to be correct as first arg to Gtk.show_uri(). But it crashes reliably, I tested that in various enviroments. I wonder if we have spotted a problem in PyGTK (or GI, for that matter). But I don't care that much, actually.

Reply:
The types of gdk.Screen() and gdk.Screen.get_default() are different:

    >>> gdk.Screen()
    <Gdk.Screen object at 0x7f7e89bd4f00 (GdkScreen at 0x56502c04eb60)>
    >>> gdk.Screen().get_default()
    <GdkX11.X11Screen object at 0x7f7e89b8d690 (GdkX11Screen at 0x56502bf36000)>

Reply:
Interesting, but the `None` version you pushed seems fine, thanks.